#!/usr/bin/env python
PACKAGE = "nmpt_saliency"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("dt", double_t, 0, "dt", 0.125, 0.001, 2.0)
gen.add("drag", double_t, 0, "drag", 0.0, 0.001, 2.0)
gen.add("move_cost", double_t, 0, "move cost", 0.015, 0.001, 0.010)
gen.add("color_saliency_image", bool_t, 0, "show debug saliency window in colors", False);

exit(gen.generate(PACKAGE, "nmpt_saliency", "nmpt_saliency"))

